cinder/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Cinder driver container for Dell EMC VNX and Unity
+# Using openstack-cinder-volume base image
+# Version 1.0.0
+
+FROM registry.access.redhat.com/rhosp12/openstack-cinder-volume:12.0-20180319.1
+
+MAINTAINER Dell EMC
+LABEL name="rhosp12/openstack-cinder-volume-dellemc"
+LABEL vendor="Dell EMC"
+LABEL version="1.0.0"
+LABEL release="12.0-20180319.1"
+LABEL storops="0.5.7"
```

Comment:
Run 'sh osp_deploy.sh download' without storops version specified, the latest version will be downloaded, maybe not 0.5.7.

Reply:
Can we not list the version of storops here?

Reply:
If we list storops version here, we can get it by docker inspect command, it is useful for issue triage, otherwise, it is hard to get storops version, we must enter running container and use rpm command to get the version info.

Reply:
What if the storops is updated and the user still sees 0.5.7 here?

Reply:
It is OK for OSP 12. We still need to think of a way of using the latest storops for OSP 13.